pkg/run/watch/install_bucket_source_and_ks.go

```diff
@@ -47,33 +56,43 @@ func SetupBucketSourceAndKS(ctx context.Context, log logger.Logger, kubeClient c
 	}
 	source := sourcev1.Bucket{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      devBucket,
-			Namespace: namespace,
+			Name:      RunDevBucketName,
+			Namespace: params.Namespace,
+			Annotations: map[string]string{
+				"metadata.weave.works/description": "This is a temporary Bucket created by GitOps Run. This will be cleaned up when this instance of GitOps Run is ended.",
```

Comment:
💡 A suggestion: maybe `this instance of GitOps Run is done.` or `this instance of GitOps Run is finished.` would work better (here and in the other place) instead of `this instance of GitOps Run is ended`? It looks like `process instance is done` or `finished` is used more often.

Reply:
They are just copied from @JamWils' texts here:

https://github.com/weaveworks/weave-gitops/issues/2813#issue-1390112050

Reply:
Ah, OK, thanks for the info.